amazon-link-engine/trunk/options.php

```diff
@@ -823,9 +823,10 @@ function getGeoriotAffiliates(suppressError) {
       margin: 30px 5px 0 0;
       font-size: 16px;
     }
-	.faq-list > li {
-		list-style-type: disc;
-	}
+    .faq-list > li {
+      list-style-type: disc;
+      margin-left: 30px;
```

Comment:
What made you decide to add a 30px margin?

Reply:
The bullets weren't indented at all, which made them look a little funky in the FAQ. Guessing the WP dashboard has some styling that resets the UL layout.